dr_mp3.h

```diff
@@ -2296,8 +2296,8 @@ DRMP3_API int drmp3dec_decode_frame(drmp3dec *dec, const drmp3_uint8 *mp3, int m
             drmp3dec_init(dec);
             return 0;
         }
-        success = drmp3_L3_restore_reservoir(dec, bs_frame, &scratch, main_data_begin);
-        if (success && pcm != NULL)
+        int result = drmp3_L3_restore_reservoir(dec, bs_frame, &scratch, main_data_begin);
```

Comment:
this breaks c89

Reply:
got it

Reply:
I moved variable declaration to the near scope beginning

Reply:
OK